arviz/tests/external_tests/test_data_pyjags.py

```diff
@@ -0,0 +1,147 @@
+                    [1.84430403, 2.39644352, 1.41523414],
+                ],
+            ]
+        ),
+        "int": np.array(
+            [
+                [
+                    [-4.44639839e-01, -2.82083775e-01, 1.65670705e-01],
+                    [-1.09586137e-01, -3.82176633e-02, -4.59948901e-01],
+                    [-1.21555094e-01, -9.03544940e-02, 1.15638469e-01],
+                    [-5.31406819e-02, 1.27035091e-01, -5.04614201e-01],
+                    [-6.11189634e-02, -1.50734274e-01, -8.75863241e-02],
+                    [-4.67516752e-01, -9.83038059e-01, -4.30877084e-02],
+                    [-9.94845141e-02, -9.23517697e-01, -5.33564824e-02],
+                    [-2.24778885e-01, -5.64131712e-01, 5.62124814e-02],
+                    [-3.58558497e-01, 4.93352091e-01, 2.41002545e-04],
+                    [-9.14217243e-02, -4.22057786e-01, 3.35735235e-01],
+                ]
+            ]
+        ),
+    }
+
+
+@pytest.fixture()
+def arviz_samples_dict_from_pyjags_samples_dict(
+    pyjags_samples_dict: tp.Dict[str, np.ndarray]
+) -> tp.Dict[str, np.ndarray]:
+    return _convert_pyjags_samples_dictionary_to_arviz_samples_dictionary(pyjags_samples_dict)
+
+
+@pytest.fixture()
+def arviz_inference_data_from_pyjags_samples_dict(
+    pyjags_samples_dict: tp.Dict[str, np.ndarray]
+) -> az.InferenceData:
+    return from_pyjags(pyjags_samples_dict)
+
+
+def verify_equality_of_numpy_values_dictionaries(
+    dict_1: tp.Dict[tp.Any, np.ndarray], dict_2: tp.Dict[tp.Any, np.ndarray]
+) -> bool:
+    if dict_1.keys() != dict_2.keys():
+        return False
+
+    for key in dict_1.keys():
+        if not np.all(dict_1[key] == dict_2[key]):
+            return False
+
+    return True
+
+
+def test_convert_pyjags_samples_dictionary_to_arviz_samples_dictionary(
+    pyjags_samples_dict: tp.Dict[str, np.ndarray],
+    arviz_samples_dict_from_pyjags_samples_dict: tp.Dict[str, np.ndarray],
+):
+    pyjags_samples_dict_from_arviz_samples_dict_from_pyjags_samples_dict = _convert_arviz_samples_dictionary_to_pyjags_samples_dictionary(
+        arviz_samples_dict_from_pyjags_samples_dict
+    )
+
+    assert verify_equality_of_numpy_values_dictionaries(
+        pyjags_samples_dict, pyjags_samples_dict_from_arviz_samples_dict_from_pyjags_samples_dict,
+    )
+
+
+def test_extract_samples_dictionary_from_arviz_inference_data(
+    arviz_inference_data_from_pyjags_samples_dict: az.InferenceData,
+    arviz_samples_dict_from_pyjags_samples_dict: tp.Dict[str, np.ndarray],
+):
+    arviz_samples_dict_from_arviz_inference_data_from_pyjags_samples_dict = _extract_samples_dictionary_from_arviz_inference_data(
+        arviz_inference_data_from_pyjags_samples_dict
+    )
+
+    assert verify_equality_of_numpy_values_dictionaries(
+        arviz_samples_dict_from_pyjags_samples_dict,
+        arviz_samples_dict_from_arviz_inference_data_from_pyjags_samples_dict,
+    )
+
+
+def test_from_pyjags(
```

Comment:
I does not look like it is testing `from_pyjags`

Reply:
It is intended to be a roundtrip test. It starts off with a PyJAGS dictionary, which is converted to InferenceData using `from_pyjags` (in the fixture arviz_inference_data_from_pyjags_samples_dict) and then converted back into a PyJAGS dictionary before being compared to the original data.

Reply:
Ohh, I see now, sorry. Maybe it can be renamed to `test_from_pyjags_round` or something like that? 

Also, I think it is even more important to test that the generated inference data has everything it should have. We have a helper function [check_multiple_attrs](https://github.com/arviz-devs/arviz/blob/master/arviz/tests/helpers.py#L169) to ease this kind of checks which is used in the other tests on converter functions

Reply:
Ok, I took care of it